discovery-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+spring:
+  application:
+    name: discovery-server
+
+server:
+  port: 8761
```

Comment:
need to decide.do wekeep 8761? or 8080.
for local i guess  8761 makes sense. so we can go with it also for not local deployment.
but then the dockerfile needs to be changed